requirements.in

```diff
@@ -75,7 +75,6 @@ Wand
 WeasyPrint
 webp
 websockets
-xlrd
```

Comment:
Why? From what I can tell, pandas still needs `xlrd` to read `.xls` files

Reply:
xlrd isn't really maintained anymore, and doesn't support Python 3.12. I could only find us using pandas in some internal, donation related tool, where we probably don't encounter old `.xls` files. Do we rely on pandas' Excel features indirectly in other places?

Reply:
Okay. I think we should be good. Was just wondering why this was in here